src/main/resources/view/ResultDisplay.css

```diff
@@ -22,3 +22,9 @@
     -fx-background-color: transparent;
 }
 
+#resultDisplay .scroll-bar {
+    -fx-opacity: 0;
+    -fx-pref-width: 1;
+    -fx-max-width: 0;
+    -fx-min-width: 0;
+}
```

Comment:
Perhaps we can modify the scrollbar scrolling speed to be slightly slower to allow for a more pleasant user experience.